src/controllers/__tests__/events.test.js

```diff
@@ -334,6 +334,25 @@ describe('Testing events for routes require auth controls', () => {
     });
   });
 
+  it('PATCH /events/:id should update the event', async () => {
+    const ngo = await createNgo();
+    const ngoCookie = createToken(ngo);
+    events[0].ngo = ngo._id;
```

Comment:
Great work! 
test has passed, and good job keeping consistency with other tests.